db/schema.rb

```diff
@@ -13,225 +12,222 @@
 
 ActiveRecord::Schema.define(version: 20160610135455) do
 
-  create_table "announcements", force: :cascade do |t|
-    t.text     "message",    limit: 65535
-    t.date     "starts_at"
-    t.date     "ends_at"
+  create_table "announcements", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
Actually, it might be a Rails 5.1 thing - http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-create_table